packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -529,6 +532,14 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         }
     }, [hideNonMatches]);
 
+    /*
+     * React on changes to the showExternalIDs prop.
+     */
+    useEffect(() => {
+        if (showExternalIDs) {
```

Comment:
Won't this only pass if the value is true-ish? What if you set it to false runtime?

Reply:
Yes, it will only pass when showExternalIDs is true. And I believe this is only what we care for. Anything else than true, should not show external ID.

Reply:
One of the fundamental ideas of the Map Template is that you should be able to switch prop values runtime. This breaks with that principle. @andreeaceachir142 any opinion here?

Reply:
@ammapspeople @matbmapspeople I agree with Anders here, that's out unwritten principle for the Map Template. 
I think it might be worth looking into the effort to make this work. 
On the other hand, there are a few other props that do not work runtime, but that is not a pattern that we want to follow.

Reply:
Fixed @ammapspeople. Now we can read true/false values from the URL.